lib/condor.py

```diff
@@ -260,7 +262,27 @@ def load_submit_file(filename: str) -> Tuple[Any, Optional[int]]:
     return htcondor.Submit(res), nqueue
 
 
-# pylint: disable=dangerous-default-value,too-many-locals,too-many-branches
+def ran_vault_storer_recently(schedd: str, handle: str, outbase: str) -> bool:
+    """See if we left a timestamp file from a previous run of jobsub_submit,
+    and it was within the last week minus a bit"""
+    try:
+        sv = os.stat(f"{outbase}/.success_vt_store_{schedd}_{handle}")
```

Comment:
One more thing I just thought of.  Since we use this path (`f"{outbase}/.success_vt_store_{schedd}_{handle}"`) in multiple places, what would you think of having a function that returns the filename based on the outbase, schedd, and handle?

Something like:

```python
def some_name(schedd, handle, outbase):
  return f"{outbase}/.success_vt_store_{schedd}_{handle}"
```

And then call that function in `ran_vault_storer_recently` and `record_vault_storer_run`? 

The upside would be that if we ever changed the naming convention, we'd only make the change in one place.  The downside of course is it slightly complicates the functions here (e.g. `os.stat(some_name(schedd, handle, outbase))`).

Reply:
Did you read my mind?!?